apps/ui-layout/registry/components/accordion/faq.tsx

```
@@ -15,80 +15,91 @@ const tabs = [
             <motion.div
               key={tab.title}
               className={`overflow-hidden ${index !== tabs.length - 1 ? 'border-b' : ''}`}
-              onClick={() => handleClick(index)}
             >
+              {/* Accessible button - handle click here instead of the wrapping div */}
               <button
-                className={`p-3 px-2 w-full cursor-pointer sm:text-base text-xs items-center transition-all font-semibold dark:text-white text-black   flex gap-2 
-               `}
+                onClick={() => handleClick(index)}
+                className='p-3 px-2 w-full cursor-pointer sm:text-base text-xs items-center transition-all font-semibold dark:text-white text-black flex gap-2'
               >
```

Biome (2.4.15)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)